### PR TITLE
Shared SKU fixes

### PR DESCRIPTION
**Pull request by giotiskl**
**Done**
---
* Shopify now relies on internal variant IDs instead of SKUs
* Client secret removed from Commerce Layer
* CSS overflow problems fixed for all 3 ecommerce vendor apps.

### CODE REVIEW

File: marketplace/commercelayer-products/src/index.js [hunk 1]
```diff
@@ -74,10 +41,10 @@ async function fetchSKUs(installationParams, search, pagination) {
     throw new Error(validationError);
   }
 
-  const { clientId, apiEndpoint, clientSecret } = installationParams;
-  const auth = await CLayerAuth.integration({
+  const { clientId, apiEndpoint } = installationParams;
+  const { accessToken } = await CLayerAuth.getIntegrationToken({
```
Comment (jelz) [inline marketplace/commercelayer-products/src/index.js]: This happens twice. Can you:

- extract to a function
- add a comment mentioning why we pass an empty string, mentioning fixed version of the SDK

File: marketplace/shopify-sku/src/productResolvers.js [hunk 2]
```diff
@@ -29,15 +29,25 @@ export const fetchProductPreviews = async (skus, config) => {
...
+  const res = await window.fetch(`https://${config.apiEndpoint}/api/2019-10/graphql`, {
+    method: 'POST',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      'x-shopify-storefront-access-token': config.storefrontAccessToken
+    },
+    body: JSON.stringify({
+      query: `{nodes (ids: [${skus
+        .map(sku => `"${sku}"`)
```
Comment (jelz) [inline marketplace/shopify-sku/src/productResolvers.js]: Compute the string w/ IDs outside of this string template

File: marketplace/shopify-sku/src/productResolvers.js [hunk 3]
```diff
@@ -29,15 +29,25 @@ export const fetchProductPreviews = async (skus, config) => {
...
+        .join(
+          ','
+        )}]) { id, __typename ...on ProductVariant { sku, image { src: originalSrc }, title, product { id, title } } }}`
```
Comment (jelz) [inline marketplace/shopify-sku/src/productResolvers.js]: how about some line breaks?

File: marketplace/commercelayer-products/src/index.js (hunk 1 above)
Comment (giotiskl) [reply marketplace/commercelayer-products/src/index.js]: Done, actually on top of that I make sure we fetch only one token, no need to fetch a new token for every req we issue

File: marketplace/shopify-sku/src/productResolvers.js (hunk 2 above)
Comment (giotiskl) [reply marketplace/shopify-sku/src/productResolvers.js]: Done.

File: marketplace/shopify-sku/src/productResolvers.js (hunk 3 above)
Comment (giotiskl) [reply marketplace/shopify-sku/src/productResolvers.js]: done.

**Comment by giotiskl** [comment]
> @giotiskl package-lock.json change seems a little bit too big. Can you do lerna bootstrap and then `--save-exact` for the new SDK version

Did it, no change

**Pull request merged**